scripts/protocgen.sh

```diff
@@ -26,12 +26,12 @@ Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
 done
 
 # command to generate docs using protoc-gen-doc
-buf protoc \
-  -I "proto" \
-  -I "third_party/proto" \
-  --doc_out=./docs/core \
-  --doc_opt=./docs/protodoc-markdown.tmpl,proto-docs.md \
-  $(find "$(pwd)/proto" -maxdepth 5 -name '*.proto')
+# buf protoc \
+#   -I "proto" \
+#   -I "third_party/proto" \
+#   --doc_out=./docs/core \
+#   --doc_opt=./docs/protodoc-markdown.tmpl,proto-docs.md \
+#   $(find "$(pwd)/proto" -maxdepth 5 -name '*.proto')
```

Comment:
shoudl we undo this? or is the fix here to remove this completely?

Reply:
I'll confirm what the fix is and lyk